scripts/datasets/information_need_annotation/information_need_annotator.py

```diff
@@ -0,0 +1,89 @@
+"""Annotate information need based on annotated dialogue.
+
+This annotator uses the annotated dialogue to identify a possible information
+need that the user had during the conversation. The information need comprises
+constraints and requests that user expressed during the conversation.
+"""
+
+import json
+import logging
+import os
+
+import yaml
+from ollama import Client, Options
+
+from dialoguekit.core.dialogue import Dialogue
+from usersimcrs.core.information_need import InformationNeed
+
+DEFAULT_INITIAL_PROMPT_FILE = "scripts/datasets/information_need_annotation/information_need_prompt_default.txt"  # noqa: E501
+
+
+class InformationNeedAnnotator:
+    def __init__(
+        self,
+        configuration_file: str,
+        prompt_file: str = DEFAULT_INITIAL_PROMPT_FILE,
+    ) -> None:
+        """Initializes the annotator.
+
+        Args:
+            configuration_file: Configuration file for Ollama.
```

Comment:
Nit: if it's Ollama specific then consider renaming to ollama_config_file

Reply:
Good point, done.